File: Programmers/Lv0/머쓱이보다키큰사람.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> array, int height) {
    array.push_back(height);
    sort(array.begin(), array.end());
    reverse(array.begin(), array.end());
    auto it = find(array.begin(), array.end(), height);
    return distance(array.begin(), it);
}

File: Programmers/Lv0/머쓱이보다키큰사람.py
def solution(array, height):
    cnt = 0
    for heig in array:
        if height < heig:
           cnt += 1 
    return cnt

# 삽입과 정렬을 통한 방법
def solution(array, height):
    array.append(height)
    array.sort(reverse=True)
    return array.index(height)